src/main/java/runge/wordledictionary/WordleDictionary.java

```diff
@@ -7,27 +7,21 @@
 {
     private ArrayList<String> definitionSplit;
     private ArrayList<String> wordSplit;
-    private final File dictionary;
 
     private final BufferedReader bufferedReader;
 
-    public File getDictionary()
-    {
-        return dictionary;
-    }
 
     public WordleDictionary() throws IOException
     {
         wordSplit = new ArrayList<>();
         definitionSplit = new ArrayList<>();
 
-        dictionary = new File("src/main/java/runge/dictionary/dictionary.txt");
+        File dictionary = new File("src/main/java/runge/wordledictionary/dictionary.txt");
         bufferedReader = new BufferedReader(new FileReader(dictionary));
-        //keep the dictionary open
-        //aka get all the objects that are being called to getDefinition of and then find them all at once
+
         if (dictionary.exists())
         {
-            for (int i = 167964; i > 0; i--)
+            for (int i = 0; i < 167964; i++)
```

Comment:
Can you think of a better way of doing this without hardcoding 167964 into your code?
Hint: https://docs.oracle.com/javase/8/docs/api/java/io/BufferedReader.html#readLine--